test/parallel/test-debug-signal-cluster.js

```diff
@@ -23,10 +23,12 @@ var common = require('../common');
 var assert = require('assert');
 var spawn = require('child_process').spawn;
 
-var args = [ common.fixturesDir + '/clustered-server/app.js' ];
-var child = spawn(process.execPath, args, {
-  stdio: [ 'pipe', 'pipe', 'pipe', 'ipc' ]
-});
+var port = common.PORT + 42;
+var args = ['--debug-port=' + port,
+            common.fixturesDir + '/clustered-server/app.js'];
+var options = {stdio: ['inherit', 'inherit', 'pipe', 'ipc']};
```

Comment:
Out of curiosity, what is the reasoning behind this change?

```diff
- stdio: [ 'pipe', 'pipe', 'pipe', 'ipc' ]
+ stdio: ['inherit', 'inherit', 'pipe', 'ipc']
```

Also IIRC the convention is to put a space after an opening curly brace and before a closing curling brace.

```js
{ stdio: ['inherit', 'inherit', 'pipe', 'ipc'] }
```